src/deferred.js

```diff
@@ -41,7 +42,7 @@ function adoptValue( value, resolve, reject ) {
 
 		// Support: Android 4.0 only
 		// Strict mode functions invoked without .call/.apply get global-object context
-		reject.call( undefined, value );
+		reject.apply( undefined, [ value ] );
```

Comment:
Consistency with the preceding block, which also effects size reduction (reuse of the `.apply( undefined, [ value ]…` pattern is good for compression).